project/ScalafixBuild.scala

```diff
@@ -212,6 +213,21 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
         s"unit/testOnly -- -l scalafix.internal.tests.utils.SkipWindows" ::
         s
     },
+    commands += Command.command("ci-explicit-result-types-rule") { s =>
```

Comment:
Instead of doing a new command that loops through all binary versions, maybe we could patch existing `ci-212` & `ci-213` commands to run an extra `unit/test scalafix.tests.rule.RuleSuite` for the N-1 & N-2 versions (since N is already run as part of the `unit/test`)? I think CI failures there would be clearer to troubleshoot (and everything would be already compiled)